### DISCUSSION TITLE
Icon for msi

### DISCUSSION THREAD

**Discussion by pankajmjx** [marcelotduarte/cx_Freeze#1984] [Q&A]
Currently there is an option to add icon to executable(.exe) which is generated after installation of msi using below option -
```
 exe = Executable(script='xxx',
                 base=None,
                 target_name="xyz.exe",
                 icon = 'xyz.ico'
                 )
```
Is there any way to add icon to msi setup file itself?

**Comment by marcelotduarte** [comment]
Check this sample: https://github.com/marcelotduarte/cx_Freeze/tree/main/samples/msi_binary_data

**Reply by pankajmjx** [reply]
This did not work.I tried running this sample locally ,the icon of msi generated is still default one as shown in screenshot.Is there anything extra i need to do along with this sample to replace this default icon?


![Screenshot (184)](https://github.com/marcelotduarte/cx_Freeze/assets/65227324/8184b4eb-5434-4ddf-a78e-798fa6490a4d)

**Reply by marcelotduarte** [reply]
Perhaps [install_icon](https://cx-freeze.readthedocs.io/en/stable/setup_script.html#cmdoption-arg-install_icon).
